src/util/sharing_map.h

```diff
@@ -160,7 +160,7 @@ class sharing_mapt
   node_type map;
 
   // number of elements in the map
-  size_type num=0;
+  size_type num = 0;
```

Comment:
The changes in this file all seem to be code style changes unrelated to what the commit message says this commit is about.